src/github.rs

```diff
@@ -38,7 +38,7 @@ pub struct Releases {
     pub tarball_url: String,
     #[serde(rename = "zipball_url")]
     pub zipball_url: String,
-    pub body: String,
+    pub body: Option<String>,
```

Comment:
this one and the previous ones will need either to make sure we make all fields `Option` (as they have changed with no warning  in the last couple of weeks) or autogenerate this based on some codegen we can run at build time, but then we risk having non-compiling code.... created #35 for this